doc/input-output-reference/src/overview/group-zone-equipment.tex

```diff
@@ -358,3 +362,193 @@ \subsubsection{Inputs}\label{inputs-2-048}
+
+\paragraph{Field: Space Air Exhaust Node or NodeList Name}\label{field-space-air-exhaust-node-or-nodelist-name}
+
+List of exhaust nodes leaving the space for exhaust fans, space energy recovery, etc. However these nodes are also used as sources of space air for space components such as fan coil units, unit heaters and ventilators, and window air conditioners. For each such component attached to a space there should be a unique space exhaust node acting as the inlet node to the component. If there is only one node -- its name can be put in this field. If there is more than one node, this must be the name of a node list object (a node list object can also contain only one node name). If there are no air exhaust nodes, this field should be blank.
+
+\paragraph{Field: Space Air Node Name}\label{field-space-air-node-name}
+
+The conditions at this node represent the average state of the air in the space. For spaces modeled as fully mixed the conditions at this node are assumed to represent the air in the entire space. This field is required for all SpaceHVAC:EquipmentConnections statements.
+
+\paragraph{Field: Space Return Air Node or NodeList Name}\label{field-space-return-air-node-or-nodelist-name}
+
+The name of one or more return air nodes which attach the space to an air loop return air path. The conditions at each return node represent the state of the air leaving the space including any heat gain assigned to that return node from \hyperref[lights-000]{Lights}, refrigeration equipment or airflow windows.
+
+\paragraph{Field: Space Return Air Node 1 Flow Rate Fraction Schedule Name}\label{field-space-return-air-flow-rate-fraction-schedule-name}
+
+The name of a schedule to specify the return air flow rate for the first return air node as a fraction of the base return air. If the next field is blank, then the return air flow rate is the total supply inlet flow rate to the space less the total exhaust node flow rate from the space multiplied by this schedule name. If this field is left blank, the schedule defaults to 1.0 at all times.
+
+\paragraph{Field: Space Return Air Node 1 Flow Rate Basis Node or NodeList Name}\label{field-space-return-air-flow-rate-basis-node-or-nodelist-name}
+
+The name of a node or list of nodes (\hyperref[nodelist]{NodeList}) that is used to calculate the return air flow rate for the first return air node in this space. The sum of the current flow rates for this node(s) multiplied by the Space Return Air Node 1 Flow Rate Fraction Schedule determines the return air flow rate. If this field is blank, then the base return air flow rate is the total supply inlet flow rate to the space less the total exhaust node flow rate from the space in the case of a single air loop serving this space. If there are multiple air loops serving this space, the base return air flow rate is governed by the corresponding supply inlet flow rate and the AirloopHVAC Design Return Air Flow Fraction of Supply Air Flow.
```

Comment:
I assume this node name is the name of a space inlet node? When would this node not be a space inlet node?

Reply:
Probably but not limited to that.